src/views/accounts/AccountBackup/Executions/AccountBackupExecutionList.vue

```diff
@@ -61,10 +61,10 @@ export default {
             formatter: DetailFormatter,
             formatterArgs: {
               drawer: true,
-              getTitle: ({ row }) => row.snapshot.name,
+              getTitle: ({ row }) => row.automation.name,
               getRoute: ({ row }) => ({
                 name: 'AccountBackupDetail',
-                params: { id: row.automation }
+                params: { id: row.automation.id }
               })
             },
             id: ({ row }) => row.automation
```

Comment:
The code does not contain any inconsistencies, issues, or suggestions for optimization as of October 2023. It appears to be structured correctly without errors or missing functionalities from the original September 2021 version. No changes should be expected.

Please note that since this is an older codebase, it's important to review security and performance risks based on current coding standards, especially regarding handling user information (like names) more securely than what was done initially due to privacy concerns.